src/config.c

```diff
@@ -422,6 +423,13 @@ static RRStatus processConfigParam(const char *keyword, const char *value, Redis
         if (strlen(value) > 0) {
             target->cluster_user = RedisModule_Strdup(value);
         }
+    } else if (!strcmp(keyword, CONF_SCAN_SIZE)) {
+        if (strlen(value) > 0) {
```

Comment:
Please handle invalid values: `goto invalid_value;` + i think it's better to fix `CONF_CLUSTER_USER` and `CONF_CLUSTER_PASSWORD` too.

Reply:
https://github.com/RedisLabs/redisraft/pull/416